docs/content/src/learn/tutorials/messaging-integrations/messaging-with-kafka/src/product_management_system/consumer2.bal

```diff
@@ -0,0 +1,35 @@
+import ballerina/kafka;
+import ballerina/io;
+import ballerina/lang.'string as strings;
+
+// Kafka Consumer Configuration
+kafka:ConsumerConfig consumer2 = {
+    bootstrapServers: "localhost:9092",
+    groupId: "consumer",
+    topics: ["product-price"],
+    pollingIntervalInMillis: 1000,
+    partitionAssignmentStrategy: "org.apache.kafka.clients.consumer.RoundRobinAssignor"
+};
+
+// Kafka Listener
+listener kafka:Consumer productConsumer2 = new (consumer2);
```

Comment:
Rename `productConsumer2` to a meaningful name.

Reply:
Done